### PR TITLE
WIP - Removed -u flag from 'go get' in install-tools

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This might be what's causing the updates to go.mod/go.sum after `./.ci/script.sh` is executed.

**Comment by objectiser** [comment]
@jpkrohling I tried out this change on latest master (built with operator sdk 0.10.0) and it still changed the go files. Also got:
```
go: finding golang.org/x/lint latest
FATA[0000] Project not in $GOPATH                       
make: *** [Makefile:235: generate] Error 1
Failed to build the operator.
```

File: Makefile [hunk 1]
```diff
@@ -255,7 +255,7 @@ install-sdk:
 
 .PHONY: install-tools
 install-tools:
-	@${GO_FLAGS} go get -u \
```
Comment (pavolloffay) [inline Makefile]: How does this affect CI, the downloaded tools are not present in the filesystem.

In the main repository we faced an issue with inconsistent tools version. The CI had always new version but local version ware outdated due to missing `-u`.

**Comment by jpkrohling** [comment]
I'm adding `WIP` to the title, as it's not doing what I think it's doing. I thought it would not touch go.mod/go.sum, but looks like it was just a coincidence that this did not happen locally when I tried this out first.

File: Makefile (hunk 1 above)
Comment (jpkrohling) [reply Makefile]: CI will always download the tool. The `-u` is only about updating the libraries that are being used. I thought this meant *also* the ones specified in the current directory's `go.mod`/`go.sum`, but I was wrong. The doc says:

> The -u flag instructs get to use the network to update the named packages and their dependencies. By default, get uses the network to check out missing packages but does not use it to look for updates to existing packages.

The purpose of this PR is to get the builds to not change go.mod/go.sum, unless explicitly told to do so.

**Comment by jpkrohling** [comment]
In time: go.mod/go.sum seems to be updated because of `go list`. Why on earth would `go list` update `go.mod` is beyond me...

**Pull request closed without merging**